components/object_store/src/disk_cache.rs

```
@@ -169,14 +169,23 @@ impl DiskCache {
         self.update_cache(filename, None).await
     }
 
-    async fn get(&self, key: &str) -> Result<Option<Bytes>> {
+    async fn get(&self, key: &str) -> Option<Bytes> {
```

Do we need to ignore error for other methods, like `insert`, `recover`?

---

Yes. It should be.